src/main/java/cotuba/CommandLineOptions.java

```diff
@@ -0,0 +1,24 @@
+package cotuba;
+
+import org.apache.commons.cli.Option;
+
+public enum CommandLineOptions {
+    DIR(new Option("d", "dir", true,
+            "Diretório que contem os arquivos md. Default: diretório atual.")),
+    FORMAT(new Option("f", "format", true,
+            "Formato de saída do ebook. Pode ser: pdf ou epub. Default: pdf")),
+    OUTPUT(new Option("o", "output", true,
+            "Arquivo de saída do ebook. Default: book.{formato}.")),
+    VERBOSE(new Option("o", "output", true,
+            "Arquivo de saída do ebook. Default: book.{formato}."));
```

Comment:
Peguei um bug aqui

```suggestion
    VERBOSE(new Option("v", "verbose", false, "Habilita modo verboso."));
```

Option opcaoModoVerboso = new Option("v", "verbose", false,	
                "Habilita modo verboso.");

Reply:
Ah sim, um dos problemas de usar CTRL+ C e CTRL+ V sem prestar atenção, vou corrigir.

Reply:
Esse problema é um clássico! Hehe